### PR TITLE
Merge latest changes from rate_limiting to master

### PR DESCRIPTION
**Pull request by mangchiandjjoe**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
   - stable
 
-lang: go
-
-go:
-  - 1.7.x
+language: cpp
 
 jdk:
   - oraclejdk8
```
Comment (qiwzhang) [inline .travis.yml]: These changes are not part of rate limiting.   Could you merge master to your branch first to get rid of these changes?

Comment (mangchiandjjoe) [inline .travis.yml]: Sure I will

**Comment by mangchiandjjoe** [comment]
The Travis CI build failed was failed with following message.

ERROR: /home/travis/build/istio/proxy/src/envoy/mixer/integration_test/BUILD:21:1: no such package '@com_github_istio_mixer//pkg/tracing': no such package '@io_bazel_rules_go_repository_tools//': Traceback (most recent call last):
	File "/home/travis/bazel/outbase/external/io_bazel_rules_go/go/private/go_repositories.bzl", line 85
		_fetch_repository_tools_deps(ctx, goroot, gopath)
	File "/home/travis/bazel/outbase/external/io_bazel_rules_go/go/private/go_repositories.bzl", line 54, in _fetch_repository_tools_deps
		ctx.download_and_extract('%s/archive/%s.zip' % (dep.repo,...), <4 more arguments>)
java.io.IOException: Prefix buildifier-81c36a8418cb803d381335c95f8bb419ad1efa27 was given, but not found in the zip and referenced by '//src/envoy/mixer/integration_test:go_default_library'.
ERROR: Analysis of target '//src/envoy/mixer/integration_test:quota_test' failed; build aborted.


I checked the master branch, it has the same build error.


$ bazel build ...:all
...........
ERROR: /usr/local/google/home/jaebong/merge/test/proxy/src/envoy/mixer/integration_test/BUILD:21:1: no such package '@com_github_gogo_protobuf//types': no such package '@io_bazel_rules_go_repository_tools//': Traceback (most recent call last):
	File "/usr/local/google/home/jaebong/.cache/bazel/_bazel_jaebong/91a20f4481b448b1e81897ac132eaa27/external/io_bazel_rules_go/go/private/go_repositories.bzl", line 85
		_fetch_repository_tools_deps(ctx, goroot, gopath)
	File "/usr/local/google/home/jaebong/.cache/bazel/_bazel_jaebong/91a20f4481b448b1e81897ac132eaa27/external/io_bazel_rules_go/go/private/go_repositories.bzl", line 54, in _fetch_repository_tools_deps
		ctx.download_and_extract('%s/archive/%s.zip' % (dep.repo,...), <4 more arguments>)
java.io.IOException: Prefix buildifier-81c36a8418cb803d381335c95f8bb419ad1efa27 was given, but not found in the zip and referenced by '//src/envoy/mixer/integration_test:go_default_library'.
ERROR: Analysis of target '//src/envoy/mixer/integration_test:go_default_library' failed; build aborted.
INFO: Elapsed time: 141.458s

**Comment by qiwzhang** [comment]
Please ignore Travis CI tests for now.  I need to figure out why.  -Wayne

**Comment by mangchiandjjoe** [comment]
Ok I got it

**Pull request merged**